CHANGELOG.rst

```diff
@@ -1,6 +1,27 @@
 Changelog
 =========
 
+0.3.0 (2018-07-18)
+------------------
+
+Features:
+
+- Spark MLlib integration, we now support logging SparkML Models directly in the log_model API, Model format, and serving APIs (#72, @tomasatdatabricks)
```

Comment:
Nit: instead of "integration," it would be more grammatically correct to write "integration:"